nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/preliminary/HeapMemoryUsageHealthCheck.java

```diff
@@ -91,7 +91,7 @@ public HealthCheckResult doCheck() {
         return result;
     }
 
-    private Double calculatePercentage(MemoryUsage usage) {
+    private static Double calculatePercentage(MemoryUsage usage) {
```

Comment:
Shouldn't this method be moved to class MemoryUsage as an instance method?

Reply:
In principle this could be better but here we have no choice as this is `java.lang.management.MemoryUsage`.

Reply:
Haha, sorry. I only read the diff and didn't know about this class.